Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -44,7 +44,6 @@ Check out the [license page](LICENSE) for more information.
   <ItemGroup Label="Package files">
     <None Include="..\FluentAssertions.png" Pack="true" Visible="false" PackagePath="" />
     <None Include="..\..\LICENSE" Pack="true" PackagePath=""/>
-    <None Include="..\init.ps1" Pack="true" PackagePath="tools"/>
```

Comment:
Should we also _remove_ `init.ps1`?
`init.ps1` seems overly complicated to open an url.
See https://github.com/fluentassertions/fluentassertions/pull/2943#discussion_r1918909066